eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,50 @@
 param (
-  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
-  $RootDirectory, # ideally $(Build.SourcesDirectory)
+  $PathToOwners, # should be in relative form from root of repo. EG: sdk/servicebus
+  $ToolVersion = "", # Placeholder. Will update in next PR
+  $ToolPath = "$env:AGENT_TOOLSDIRECTORY", # The place to check the tool existence. Put $(Agent.ToolsDirectory) as default
+  $DevOpsFeed = "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json", # DevOp tool feeds.
+  $WorkingDirectory = "$env:SYSTEM_DEFAULTWORKINGDIRECTORY", # The place we fetch CODEOWNERS file
   $VsoVariable = "" # target devops output variable
 )
-$target = $TargetDirectory.ToLower().Trim("/")
-$codeOwnersLocation = Join-Path $RootDirectory -ChildPath ".github/CODEOWNERS"
-$ownedFolders = @{}
 
-if (!(Test-Path $codeOwnersLocation)) {
-  Write-Host "Unable to find CODEOWNERS file in target directory $RootDirectory"
-  exit 1
-}
-
-$codeOwnersContent = Get-Content $codeOwnersLocation
+$EngOutputIndicator = ".*EngToolOutputProperty: (.*)"
```

Comment:
We should move this into the function doing the matching. Also we need to make sure that the regex match allows for multiple lines as I don't think it does by default.